app/uk/gov/hmrc/trusts/connector/DesConnector.scala

```diff
@@ -40,8 +40,11 @@ import uk.gov.hmrc.trusts.utils.Constants._
 class DesConnectorImpl @Inject()(http: WSHttp, config: AppConfig) extends DesConnector {
 
   lazy val trustsServiceUrl : String = s"${config.desUrl}/trusts"
+  lazy val estatesServiceUrl : String = s"${config.desUrl}/estates"
+
   lazy val matchEndpoint : String = trustsServiceUrl + "/match"
-  lazy val registrationEndpoint : String = trustsServiceUrl + "/registration"
+  lazy val trustRegistrationEndpoint : String = trustsServiceUrl + "/registration"
+  lazy val registrationEstatesEndpoint : String = estatesServiceUrl + "/registration"
```

Comment:
rename as estateRegistrationEndpoint for consistency

Reply:
@jassalrichy  Done.